executor/mock/jobrunner.go

```diff
@@ -66,6 +67,8 @@ type JobInput struct {
 	MetatronEnabled bool
 	// Mem sets the memory resource attribute in MiB
 	Mem *int64
+	// ShmSize sets the shared memory size of `/dev/shm` in MiB
+	ShmSize *int64
```

Comment:
why int64?

Reply:
Was trying to be consistent with docker's ShmSize - though an int probably makes more sense.